src/panels/Panels.tsx

```diff
@@ -25,6 +26,8 @@ export default function Panels() {
         return <ConfigPanel />;
       case "audit-log-filters":
         return <AuditLogsFilterPanel />;
+      case "add-secret":
+        return <AddSecret />;
```

Comment:
I see that we use the "Panel" suffix for the names of all the previous panels. For consistency reasons, maybe it makes sense to name this something like "AddSecretPanel"?

Reply:
Done.